docker-compose.yml

```diff
@@ -86,6 +87,36 @@ services:
         timeout: 2s
         retries: 30
 
+  pbs:
+    image: openpbs:23.06.06
+    privileged: true
+    environment:
+      OIDC_CERT_URLS: '["http://192.168.240.3:8080/auth/realms/kcrealm/protocol/openid-connect/certs"]'

```

Comment:
Just a question, why do we use `OIDC_CERT_URLS` in this container?

Reply:
Not sure. I used the `slurm` section in the `docker-compose.yml` as example. https://github.com/eth-cscs/firecrest-v2/blob/1620f1485628196721fc46dfffa463a7ca9360e5/docker-compose.yml#L66